dll/win32/browseui/shellfind/shellfind.h

```diff
@@ -31,12 +31,58 @@
 #define WM_SEARCH_ADD_RESULT     WM_USER + 2
 #define WM_SEARCH_UPDATE_STATUS  WM_USER + 3
 
+typedef struct tagLOCATIONITEM
+{
+    struct tagLOCATIONITEM *pNext;
+    WCHAR szPath[ANYSIZE_ARRAY];
+} LOCATIONITEM;
+
+void FreeList(LOCATIONITEM *pLI);
```

Comment:
I wonder whether the FreeList function, could be made as a destructor for the LOCATIONITEM structure? (a destructor that would act on the "top-level" object without recursively calling itself on the pNext items)